assets/florence/pagedown/Markdown.Editor.js

```diff
@@ -1182,9 +1203,18 @@
             style.margin = "10px";
             style.display = "inline";
             style.width = "7em";
+            style.border = "1px solid #888888";
+            style.fontFamily = "trebuchet MS, helvetica, sans-serif";
+            style.fontSize = "0.8em";
+            style.fontWeight = "bold";
+
+            var div = doc.createElement("div");
```

Comment:
Is this ES5 or under? If ES6 + can we use `let` / `const` rather than `var`?

Reply:
No idea, how do I find out?

Reply:
If it is only used in Florence then we know it will only be internal users using it on network devices, and currently, the internal internet browser is Chrome (Can check version as it will be the same as your Windows laptop as this is one of the on network devices) the version of Chrome supports ES6 so think we are safe to use let / const)

Reply:
Done